pt/README.md

```diff
@@ -0,0 +1,35 @@
+# PT
+Intel® Processor Trace (Intel PT) is an extension of Intel® Architecture that 
+captures information about software execution using dedicated hardware facilities 
+that cause only minimal performance perturbation to the software​
+being traced. ​
+
+## Build Tips
+Need prepare intel-pt.h and libipt.so for building PT related cases.
+You can get them in https://github.com/intel/libipt.
+
+## Usage
+Need libipt.so.2 and add it into LD_LIBRARY_PATH before running PT cases.
+You can build libipt.so.2 in https://github.com/intel/libipt.
+
+./branch
+Disable branch and check no TIP/FUP/TNT.
```

Comment:
Better to put this in code block, for example

```
# Disable branch and check no TIP/FUP/TNT.
./branch
```

Same for the following parts.